$OpenBSD: patch-tools_build_src_tools_stage_jam,v 1.1 2022/02/02 12:24:14 sthen Exp $

Don't skip install targets if there's <build>no in ureqs
62e7321669f66a90e2a90c8f31af34f59ebc5d7a

Index: tools/build/src/tools/stage.jam
--- tools/build/src/tools/stage.jam.orig
+++ tools/build/src/tools/stage.jam
@@ -478,6 +478,10 @@ class install-target-class : basic-target
         return [ sequence.unique $(result2) ] ;
     }
 
+    rule skip-from-usage-requirements ( )
+    {
+    }
+
     # Returns true iff 'type' is subtype of some element of 'types-to-include'.
     #
     local rule include-type ( type : types-to-include * )
